dart/.sidekick.toml

```diff
@@ -33,6 +33,14 @@ version = "0.1.0"
 'package:google_cloud_gax' = '^0.1.0'
 'package:http'             = '^1.3.0'
 
+# Generated packages
```

Comment:
I assume these are the ones that are currently referenced by other (generated) packages.

Reply:
I'm fine w/ either, I was just confirming this for my own understanding.